docs/client/basic/toc.js

```diff
@@ -48,7 +48,7 @@ var sections = [
       item("Defining templates in HTML", {id: "defining-templates"}),
       item("Template.<em>name</em>.helpers", {longname: "Template#helpers"}),
       item("Template.<em>name</em>.events", {longname: "Template#events"}),
-      item("Template.<em>name</em>.rendered", {longname: "Template#rendered"}),
+      item("Template.<em>name</em>.onRendered", {longname: "Template#onRendered"}),
```

Comment:
actually, no, there is no backwards-compatible link redirection for basic docs, you are right.

Can we ignore it please? It is not like hundreds of stackoverflow questions link to that paragraph in basic docs and @debergalis approves breaking it.